vllm/v1/outputs.py

```diff
@@ -8,7 +8,7 @@
 class SamplerOutput:
 
     # [num_reqs]
-    sampled_token_ids: List[int]
+    sampled_token_ids: torch.Tensor
```

Comment:
I think this needs further discussion, it might hurt the TP case.

Reply:
Just to double check: @tlrmchlsmth Are you ok with this change?

Reply:
Yep, this is good!